datafusion/expr/src/expr.rs

```diff
@@ -1768,9 +1778,15 @@ fn create_name(e: &Expr) -> Result<String> {
             window_frame,
             partition_by,
             order_by,
+            null_treatment,
         }) => {
             let mut parts: Vec<String> =
                 vec![create_function_name(&fun.to_string(), false, args)?];
+
+            if null_treatment.is_some() {
```

Comment:
You can use `if let Some() = null_treatment {}` pattern here. It would eliminate `.unwrap`